From 2dbe871a73cda7c4eaa82d6efab7d4b80d029520 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Thu, 6 Sep 2018 15:13:19 +0300
Subject: [PATCH 291/706] arm64: dts: ls1028a: Add link properties to enetc
 port nodes for CFP and PXP

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit c3f4a514af46b38ab3aa17291519ab85283d419b)
(cherry picked from commit d1248ef4bbc907f17fda6b28186988b7c72549b3)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 .../boot/dts/freescale/fsl-ls1028a-cfp.dts    | 32 +++++++++++++++++++
 .../boot/dts/freescale/fsl-ls1028a-pxp.dts    | 32 +++++++++++++++++++
 2 files changed, 64 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-cfp.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-cfp.dts
index 6bcbc4aac099..1b3536cc584c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-cfp.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-cfp.dts
@@ -55,3 +55,35 @@
 	aliases {
 	};
 };
+
+&enetc_port0 {
+	phy-connection-type = "internal";
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
+
+&enetc_port1 {
+	phy-connection-type = "internal";
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
+
+&enetc_port2 {
+	phy-connection-type = "internal";
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
+
+&enetc_port3 {
+	phy-connection-type = "internal";
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-pxp.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-pxp.dts
index 944c247a4594..bed6b7277850 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-pxp.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-pxp.dts
@@ -59,3 +59,35 @@
 &dp0 {
 	status = "okay";
 };
+
+&enetc_port0 {
+	phy-connection-type = "internal";
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
+
+&enetc_port1 {
+	phy-connection-type = "internal";
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
+
+&enetc_port2 {
+	phy-connection-type = "internal";
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
+
+&enetc_port3 {
+	phy-connection-type = "internal";
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
-- 
2.17.1

